csm/SystemMan.py

```diff
@@ -0,0 +1,139 @@
+import CommsMan as CommsMan
+import StorageMan as StorageMan
+import MotorMan as MotorMan
+import CameraMan as CameraMan
+import PerceptionMan as PerceptionMan
+
+from CommsMan import CommsMan
+from StorageMan import StorageMan
+from MotorMan import MotorMan
+from CameraMan import CameraMan
+from PerceptionMan import PerceptionMan
+
+# from ImageSources import ImageSource, LocalVideo, StillImage, Camera
+# from PerceptionUtils import BoundingBox
```

Comment:
I don't know if you need these here, if you do, could you uncomment them? If you don't, could you get rid of them?